### PR TITLE
Dutch translation added

### PR DESCRIPTION
**Pull request by jhuskens**
Thank you very much for this awesome card! I have added the Dutch translation.

### CODE REVIEW

File: src/languages/nl.json [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+            "useCustomButton": "Gebruik aangepaste knop",
+            "useCustomCard": "Gebruik aangepaste kaart",
+            "showButton": "Knop weergeven",
+            "preview": "Voorvertoning",
+            "hidePreview": "Verberg voorvertoning",
+            "secondaryInfo": "Bijkomende info",
+            "secondaryInfoDesc": "Dit zal onder de titel van de kaart worden weergegeven.",
+            "secondaryInfoHelper": "Gebruik Jinja2 template om bijkomende informatie te tonen",
+            "notifyInfo": "Meldingsinstellingen",
+            "notifyInfoHelper": "Het resultaat moet "True" teruggeven om te worden weergegeven.",
```
Comment (ngocjohn) [inline src/languages/nl.json:216]: Can you please correct the double quotes "true" to single? 'true'

File: src/languages/nl.json [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+            "title": "Aangepaste kaart",
+            "desc": "Yaml configuratie voor lovelace kaart"
+        },
+        "customTireBackground": {
+            "title": "Aangepaste banden achtergrond",
+            "desc": "Upload een afbeelding om de standaard achtergrond te vervangen.",
+            "info": "De afbeelding moet een vierkant zijn met een maximale resolutie van 450x450 pixels. Een transparante achtergrond is aanbevolen."
+        },
+        "customActionConfig": {
+            "title": "Knop interactie,
```
Comment (ngocjohn) [inline src/languages/nl.json:278]: add " to the end

**Review by ngocjohn — CHANGES_REQUESTED** [review]

File: src/languages/nl.json (hunk 2 above)
Comment (jhuskens) [reply src/languages/nl.json]: Apologies, I have fixed both issues.

**Comment by ngocjohn** [comment]
Thanks!

**Pull request merged**